src/lib/viewers/media/DashViewer.js

```diff
@@ -950,6 +972,17 @@ class DashViewer extends VideoBaseViewer {
         }
     }
 
+    setQuality(quality) {
+        this.cache.set('media-quality', quality, true);
+        this.selectedQuality = quality;
```

Comment:
Do we need a separate instance property or can we pull this value from `cache` each time?